ui/components/Navigator.js

```diff
@@ -780,12 +788,10 @@ class Navigator extends React.Component {
       }
     };
 
-    state.navigatorComponents.forEach(({ title, href, children, isBeta }) => {
+    navigatorComponents.forEach(({ title, href, children, isBeta }) => {
       fetchNestedPathAndTitle(path, title, href, children, isBeta);
     });
-    st.path = path;
-    return st;
-  }
+  });
```

Comment:
useEffect missing, dependency array, please add it otherwise global stack will get overflow, and will cause error, update array with appropriate dependency
```suggestion
  }, []);
```

Reply:
Updating...

Reply:
Is it done?

Reply:
Yup, I have update the dependency array.